app/Models/Image.php

```diff
@@ -191,4 +193,23 @@ public function ensureDirectoryExists()
             mkdir($destination, 0777, true);
         }
     }
+
+    /**
+     * Deletes any generated files if they exist
+     * @param bool $includeOriginal Deletes
+     */
```

Comment:
See the comment I left on `Image.php`; a function shouldn't return values if they're not useful. If the return values of this function are meaningful but to be used later, document them and when they should be used in a `@return` section in the docblock here.